sherpa/bin/conformer_rnnt/offline_asr.py

```diff
@@ -348,10 +349,21 @@ def main():
         assert num_active_paths >= 1, num_active_paths
 
     if bpe_model_filename:
-        assert token_filename is None
+        assert token_filename is None, (
+            "You need to provide either --bpe_model_filename or --token_filename parameter."
```

Comment:
Please use dash `-`.
```suggestion
            "You need to provide either --bpe-model-filename or --token-filename parameter."
```